Wiley Online Library.js

```diff
@@ -1215,6 +1222,66 @@ var testCases = [
 				"seeAlso": []
 			}
 		]
+	},
+	{
+		"type": "web",
+		"url": "http://onlinelibrary.wiley.com.libezproxy2.syr.edu/doi/10.1111/polp.12144/full",
```

Comment:
I guess we want the url here w/o your proxy, i.e.

http://onlinelibrary.wiley.com/doi/10.1111/polp.12144/full

?